subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -340,7 +340,7 @@ uint8_t ll_adv_params_set(uint16_t interval, uint8_t adv_type,
 	adv->lll.chan_map = chan_map;
 	adv->lll.filter_policy = filter_policy;
 
-#if defined(CONFIG_BT_CTLR_SCAN_REQ_NOTIFY)
+#if defined(CONFIG_BT_CTLR_SCAN_REQ_NOTIFY) && defined(CONFIG_BT_CTLR_ADV_EXT)
 	adv->lll.scan_req_notify = sreq;
```

Comment:
Override using a variable that be set in `vs_set_scan_req_reports`  when it is called.

Reply:
If it's legit setting `CONFIG_BT_CTLR_SCAN_REQ_NOTIFY=y` independently of `CONFIG_BT_CTLR_ADV_EXT`, as noted above, I would leave as it is.
When `vs_set_scan_req_reports is called`, I'm using corresponding bit in `vs_events_mask`, as said below.